docs/installation/discord/index.md

```diff
@@ -147,5 +147,5 @@ Follow the first 4 mins of this [Video Tutorial](https://youtu.be/8o25pRbXdFw) t
 Execute following command to completely remove Botkube and related resources from your cluster.
```

Comment:
I think we can change that in 0.13 too 👍 If the docs are visible, it means people still can use that version.